src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Text.txt

```diff
@@ -266,4 +266,8 @@ Error({Kind:ErrorKind.BadLanguageCode})
 "domingo enero 31"
 
 >> Text(DateTimeValue("12:34:56"),"yyyy-mm-dd hh:mm:ss")
-"1899-12-30 12:34:56"
\ No newline at end of file
+"1899-12-30 12:34:56"
+
+// This would previously throw an exception in the interpreter by the C# text formatter on the "i"
+>> Text( 12, "i" )
```

Comment:
Should this be an error at all?

![image](https://github.com/microsoft/Power-Fx/assets/44986255/2c004da8-f669-4788-b454-7736aa6cc733)